codalab/bundles/run_bundle.py

```diff
@@ -39,11 +39,14 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('time_user', float, 'Amount of time (seconds) by user.', generated=True, formatting='duration'))
     METADATA_SPECS.append(MetadataSpec('time_system', float, 'Amount of time (seconds) by the system.', generated=True, formatting='duration'))
     METADATA_SPECS.append(MetadataSpec('memory', float, 'Amount of memory (bytes) used by this run.', generated=True, formatting='size'))
+    # TODO(klopyrev): These statistics are wrong in the old worker system and are not reported in the new worker system.
+    #                 These metadata fields should be removed.
```

Comment:
This information is not accurate for mounted volumes. I don't see it useful to keep the fields around. We can add them back in if Docker does ever fix it, although I have my doubts that they even can.